packages/vike-react-zustand/src/renderer/VikeReactZustandWrapper.tsx

```diff
@@ -33,7 +33,7 @@ export default function VikeReactZustandWrapper({ pageContext, children }: VikeR
   for (const [key, store] of stores) {
     // Trick to make import.meta.env.SSR work direclty on Node.js (without Vite)
     // @ts-expect-error
-    import.meta.env ??= { SSR: true }
+    import.meta.env = { SSR: true }
```

Comment:
Doesn't work, import.meta.env is not statically evaluated in client code in dev, it would set SSR:true in browser.

Reply:
I didn't test so I may have been wrong, but I still think it should work because `import.meta.env.SSR` is statically replaced (enabling code splitting) whenever Vite transpiles code. I.e. the *runtime* value `import.met.env.SSR` is only ever used for externalized server-side.

Reply:
> import.meta.env is not statically evaluated in client code in dev

Ah, sorry, didn't read that. Ok, let's revert then.